app/build.gradle

```diff
@@ -11,49 +11,15 @@ android {
         versionCode 12
         versionName "2.2.1"
         testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
+        buildConfigField "String", "EXODUS_API_KEY", "\"$System.env.EXODUS_API_KEY\""
```

Comment:
How can we make this the new variable is available?

Reply:
Could you please elaborate the question

Reply:
Sorry, missed a couple of words here :sweat_smile: 

How can we make **sure** that the new variable is available?
In other words, how can we test this?

Reply:
Quite easy. When compiled without the key present in the environment, the `BuildConfig.EXODUS_API_KEY` will have a `null` value, otherwise, the key will be present. This class is autogenerated by Gradle on compile-time.

Class in question: `org.eu.exodus_privacy.exodusprivacy.BuildConfig`